src/components/sphere/Sphere.tsx

```diff
@@ -9,16 +9,7 @@ import { SphereProps } from "../../types/SphereProps";
 const Sphere: React.FC<SphereProps> = (props: SphereProps) => {
 	return (
 		<span
-			className="sphere"
-			style={{
-				backgroundColor: props.backgroundColor,
-				width: props.width,
-				height: props.height,
-				bottom: `${props.bottom}%`,
-				top: `${props.top}%`,
-				left: `${props.left}%`,
-				right: `${props.right}%`,
-			}}
+			className={`sphere ${props.cssClass}`}
```

Comment:
1. call this prop `size`
2. apply the above parsing function to handle the value of this prop:
```suggestion
                        className={`sphere ${parseSizeProp(props.size)}`}
```